_data/blogs.yml

```diff
@@ -137,3 +137,8 @@
   title: Interview with a FOSSASIA Contributer | Ploypiti Piyaprapan
   img: aditya-blog-2.jpg
   author_name: Aditya Dutt
+
+- url: https://anushka264230468.wordpress.com/2017/12/16/first-blog-post/
+  title: A Student’s Experience
+  img: Student Interview Image.jpg
```

Comment:
Don't leave space or use `Student-Interview-Image.jpg` or you can use small name also.